webpack.config.js

```diff
@@ -16,23 +16,27 @@ module.exports = function (env) {
         },
         mode: "write-references",
       },
+      eslint: {
+        files: "./static/js/src/**/*.{ts,tsx,js,jsx}",
+        options: { fix: !isProd },
+      },
     }),
   ];
   return {
     mode: isProd ? "production" : "development",
     entry: {
-      main: "/static/js/src/RecentListens.tsx",
-      import: "/static/js/src/LastFMImporter.tsx",
-      userEntityChart: "/static/js/src/stats/UserEntityChart.tsx",
-      userReports: "/static/js/src/stats/UserReports.tsx",
-      userPageHeading: "/static/js/src/UserPageHeading.tsx",
-      userFeed: "/static/js/src/UserFeed.tsx",
-      playlist: "/static/js/src/playlists/Playlist.tsx",
-      playlists: "/static/js/src/playlists/Playlists.tsx",
+      main: "./static/js/src/RecentListens.tsx",
```

Comment:
I might be wrong here, but I think that the absolute paths here were because the frontend code dir was being mounted in a docker container (i think, `static_builder`) in /static for the automatic builds on code changes during dev. Would that process be affected by these changes?

Reply:
Strangely, the code was mounted both to `/code/static` and `/static` in the docker-compose file:
https://github.com/metabrainz/listenbrainz-server/blob/master/docker/docker-compose.yml#L113-L114

I'm just removing the `/static` mount and pointing any path to the equivalent `/code/static` folder.
I'm choosing to keep `/code/static` rather than `/static/` because we already have other necessary files in `/code` (https://github.com/metabrainz/listenbrainz-server/blob/master/docker/Dockerfile.webpack#L6)
It also makes sure the setup it more in line with the CI setup where we copy to `/code/static`: https://github.com/metabrainz/listenbrainz-server/blob/master/docker/Dockerfile.webpack#L13


I can confirm that static_builder does its job properly and recompiles (and reruns eslint) on code changes.

Reply:
Sounds good, thanks for confirming.

Reply:
Thanks for reviewing! I feel better having had to re-re-confirm I'm not breaking anything :)